sixtyfps_compiler/generator/rust.rs

```diff
@@ -119,6 +119,8 @@ pub fn generate(doc: &Document, diag: &mut BuildDiagnostics) -> Option<TokenStre
         .collect::<Vec<_>>();
     Some(quote! {
         #[allow(non_snake_case)]
+        #[allow(clippy::match_single_binding)]
+        #[allow(clippy::deref_addrof)]
```

Comment:
I'd go away and exclude a broad category  of lints that don't make sense for generated code:

```suggestion
        #[allow(clippy::style)]
        #[allow(clippy::complexity)]
```

Or do you think we would benefit from some of these lints?

Reply:
Excellent idea. TIL clippy categories. That makes a whole lot more sense to me.